src/app/tasks/tasks.component.ts

```diff
@@ -56,7 +61,7 @@ export class TasksComponent implements OnInit {
   }
 
   toggleTaskComplete(task) {
-    this.tasksService.addTask({ ...task, completed: !task.completed }).subscribe((res) => {
+    this.tasksService.addTask({ ...task, completed: !task.completed, completedTime: this.now }).subscribe((res) => {
```

Comment:
This is not how we add the current time to a document.  We use `couchService.datePlaceholder` which then is replaced when `couchService.updateDocument` is run.

In this case I think it would be best to append `completedTime` in **TasksService** if the task is completed.

Reply:
thank you for the review @paulbert